runtime/ms_rest_azure/lib/ms_rest_azure/azure_service_client.rb

```diff
@@ -153,7 +168,7 @@ def update_state_from_get_resource_operation(request, polling_state, custom_dese
     # @param polling_state [MsRestAzure::PollingState] polling state to update.
     # @param custom_deserialization_block [Proc] custom deserialization method for parsing response.
     #
-    def update_state_from_location_header(request, polling_state, custom_deserialization_block)
+    def update_state_from_location_header(request, polling_state, custom_deserialization_block, final_state_via)
```

Comment:
should we have a default value for final_state_via here as well?

Reply:
I think it is better to add a default value here also. Added. Thanks